test/index.test.js

```diff
@@ -14,26 +14,46 @@ const rimraf = promisify(_rimraf);
 const FIXTURES_DIR = `${__dirname}/fixtures`;
 const DEFAULT_SCRIPT = 'microbundle';
 
-const times = (n, fn) => Array.from({ length: n }).map(i => fn(i));
 const join = (arr, delimiter = '') => arr.join(delimiter);
-const constant = konst => () => konst;
 
 const printTree = (nodes, indentLevel = 0) => {
-	const indent = join(times(indentLevel, constant('  ')));
+	const indent = '  '.repeat(indentLevel);
 	return join(
 		nodes
 			.filter(node => node.name[0] !== '.')
-			.map(
-				node =>
-					`${indent}${node.name}\n${
-						node.type === 'directory'
-							? printTree(node.children, indentLevel + 1)
-							: ''
-					}`,
-			),
+			.map(node => {
+				const isDir = node.type === 'directory';
+				return `${indent}${node.name}\n${
+					isDir ? printTree(node.children, indentLevel + 1) : ''
+				}`;
+			}),
 	);
 };
 
+const getBuildScript = async (fixturePath, defaultScript) => {
+	const pkgJsonPath = resolve(fixturePath, 'package.json');
+
+	try {
+		await stat(pkgJsonPath);
+	} catch (err) {
+		if (err.code === 'ENOENT') {
+			return defaultScript;
+		}
+	}
+
+	try {
```

Comment:
is this try/catch needed now?

Reply:
IMHO I think it makes the intention clear, anyway I am open to suggestions to improve it